extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/UiResource.java

```diff
@@ -78,4 +78,19 @@ public TransferHistoryPage transferHistoryPageEndpoint() {
     public AssetDto getTransferProcessAsset(@PathParam("transferProcessId") String transferProcessId) {
         return transferHistoryPageAssetFetcherService.getAssetForTransferHistoryPage(transferProcessId);
     }
+
+    @GET
+    @Path("pages/policy-page")
+    @Produces(MediaType.APPLICATION_JSON)
+    public PolicyPage policyPage() {
+        return new PolicyPage(policyApiService.getPolicies());
+    }
+
+    @POST
+    @Path("pages/policy-page")
+    @Produces(MediaType.APPLICATION_JSON)
+    public IdResponse createPolicy(UiPolicyDto policyDto) {
```

Comment:
For the createPolicy call we need a UiPolicyCreateDto, which does not contain the legacy policy nor the errors. Meaning we are cutting down the API Model for the create process to what it exactly needs to the behavior will be exactly defined.

The response for this call could actually return a UiPolicyDto, which would make testing easier